pkg/transportrequest/rfc/upload_test.go

```diff
@@ -41,7 +41,7 @@ func TestUploadRFC(t *testing.T) {
 		err := upload.Perform(&exec)
 
 		if assert.NoError(t, err) {
-			assert.Equal(t, exec.Calls, []mock.ExecCall{mock.ExecCall{Exec: "cts", Params: []string{"uploadToABAP:123456"}}})
+			assert.Equal(t, exec.Calls, []mock.ExecCall{{Exec: "cts", Params: []string{"uploadToABAP:123456"}}})
```

Comment:
Why does this have one less mock.ExecCall than before?

Reply:
the Go staticcheck of the code studio notes a warning: something like redundant type: 2 * mock.ExecCall.
codeclimate doesnt care.

Reply:
Oh yeah I just saw that too in the old code- Redundant type and that we should delete it.